.travis.yml

```diff
@@ -49,5 +49,15 @@ install:
 - pip install -U pip setuptools tox wheel
 script:
 - tox -e "${TOX_ENV}"
+deploy:
+  provider: pypi
+  user: Surgo
```

Comment:
If you want to use your account, please put here.

- [Install and configure travis CLI](https://github.com/travis-ci/travis.rb#installation)
- Get encrypted password from `travis encrypt {your pypi password}`

Reply:
I'm good with using yours. Looks like it is all set up. 😃